packages/nuxt/test/vite/utils.test.ts

```diff
@@ -111,16 +111,16 @@ describe('constructFunctionReExport', () => {
     const result2 = constructFunctionReExport(query2, entryId);
 
     const expected = `
-async function reExport(...args) {
+async function reExport0(...args) {
```

Comment:
m: Can we also have a test here to show that it does not change/wrap an export that does not match?

Reply:
That would require changing a lot of logic in the code. The here tested function `constructFunctionReExport` is only getting the input from the query parameters and creating a function for each query param.

The part, where those query params are actually added is here:
```js
const functionsToExport = flatten(Object.values(moduleInfo.exportedBindings || {})).filter(functionName =>
  asyncFunctionReExports.includes(functionName),
);
```

One thing I could do is extracting this part as a function to test this step as well.